kahuna/public/js/search/index.js

```diff
@@ -24,14 +24,26 @@ export var search = angular.module('kahuna.search', [
     'data-structure.list-factory',
     'data-structure.ordered-set-factory',
     'gr.topBar',
-    'grPanel'
+    'grPanel',
+    'ui.router'
 ]);
 
 // TODO: add a resolver here so that if we error (e.g. 401) we don't keep trying
 // to render - similar to the image controller see:
 // https://github.com/guardian/media-service/pull/478
-search.config(['$stateProvider',
-               function($stateProvider) {
+search.config(['$stateProvider', '$urlMatcherFactoryProvider',
+               function($stateProvider, $urlMatcherFactoryProvider) {
+
+    // Query params slashes are encoded that mess up our seach query when we search for e.g. `24/7`.
+    // see: https://github.com/angular-ui/ui-router/issues/1119#issuecomment-64696060
+    // TODO: Fix this
+    $urlMatcherFactoryProvider.type('Slashed', {
+        encode: val => val || '',
+        decode: val => val || '',
```

Comment:
Saying that, I don't want to return a string for undefineds, so I have updated it.